docs/usage.md

```diff
@@ -70,17 +70,17 @@ Again, by default, the group information is only used to compute co-abundances f
 
 The typical command for running the pipeline is as follows:
 
-```console
-nextflow run nf-core/mag --input samplesheet.csv --outdir <OUTDIR> --genome GRCh37 -profile docker
+```bash
+nextflow run nf-core/mag --input samplesheet.csv --outdir <OUTDIR> --host_genome GRCh37 -profile docker
```

Comment:
```suggestion
nextflow run nf-core/mag --input samplesheet.csv --outdir <OUTDIR> -profile docker
```
The `--host_genome` parameter is not required for this pipeline and thus I would omit it here and keep a minimal example to run the pipeline with default parameters.